client/components/RelatedProducts/Related/RelatedProductsList.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import RelatedProductCard from './RelatedProductCard.jsx';
+
+const RelatedProductsList = () => {
+  //get request for related product ids based on id of current product
+  //then get request for each product to product_id (for name, category, default_price) AND get request to id/styles (for sale_price and image) as well??
```

Comment:
Good plan. Adding a handleClick function that changes our global "main product" to whatever was clicked would be really awesome later in this project as well.